package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xenit/finder-ui",
-  "version": "2.5.1-alpha.18",
+  "version": "2.5.2-alpha-1",
```

Comment:
You don't need to change the version here manually, it's done by github actions after pushing the tag (which I think should be `2.5.1-alpha.19`)

Reply:
nope, there are other tags. I'm not sure this is done automatically...

Reply:
Example: tag `v2.5.1-alpha-28` has `alpha-18` in package.json
https://github.com/xenit-eu/finder-ui/blob/450b28c144c6efdfa342006cb335cc1a08cc65f2/package.json#L3

Reply:
well.. it's corrected by the GH Action and pushed to NPM, but not committed?